ci/pipeline.yml

```diff
@@ -100,6 +175,14 @@ resources:
       tag_filter: 0.*.*
       fetch_tags: true
 
+  - name: pages-release
```

Comment:
Should this be the production `src`?

Reply:
it only includes this resource if env is production, the production `src` is defined in the block above

Reply:
Nevermind. This is just to update the release.